static/css/character_card_manager.css

```
@@ -1430,11 +1442,75 @@ width: 94%;
+}
+
+.tab-content.tab-entering {
+    position: relative;
+    z-index: 2;
+    background: #fff;
+    animation: viewCurtainReveal 0.5s cubic-bezier(0.22, 1, 0.36, 1) forwards;
+    transform-origin: top center;
+    box-shadow: 0 6px 18px rgba(64, 197, 241, 0.12);
+}
+
+@keyframes viewCurtainReveal {
```

Stylelint (17.7.0)

[error] Expected keyframe name "viewCurtainReveal" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)